README_sqlite_dbs.md

```diff
@@ -7,12 +7,27 @@ This directory contains SQLite databases that make our clean tabular
 There are DBs for the versions of the MCRF OMOP CDM data, the CDM
 vocabulary, and examples for various classification tasks.
 
+### naming convention
+
+The emr DBs are named following the convention 
+'db_name.data_date.tags.sqlite3' where db_name is
```

Comment:
I would vote for a more BNF-like syntax, e.g. `<db-name> "." <data-date> ("." <tags>)? ".sqlite3"`.